core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/LazyHttpClient.java

```diff
@@ -64,56 +47,43 @@
     @GuardedBy("lock")
     private RuntimeException initException;
     @GuardedBy("lock")
-    private CloseableHttpClient httpClient;
-
-    static ApacheSender43 create() {
-        return new ApacheSender43();
-    }
-
-    private ApacheSender43() {
-    }
+    private CloseableHttpClient delegate;
 
-    @Override
-    public HttpResponse sendRequest(HttpUriRequest request) throws IOException, FriendlyException {
-        try {
-            return getHttpClient().execute(request);
-        } catch (SSLHandshakeException e) {
-            String completeUrl = "https://" + request.getURI().getHost();
-            throw SSLUtil.newSSLFriendlyException(completeUrl);
-        }
+    public static CloseableHttpClient getInstance() {
+        return INSTANCE;
     }
 
-    @Override
-    public void dispose(HttpResponse response) {
-        try {
-            if (response != null) {
-                ((CloseableHttpResponse) response).close();
-            }
-        } catch (IOException e) {
-            logger.error("Failed to send or failed to close response, exception: {}", e.toString());
-        }
+    private LazyHttpClient() {
     }
 
-    @Override
-    public HttpClient getHttpClient() {
+    private CloseableHttpClient getDelegate() {
         synchronized (lock) {
-            if (httpClient != null) {
-                return httpClient;
+            if (delegate != null) {
+                return delegate;
             }
             if (initException != null) {
                 throw initException;
             }
             try {
-                httpClient = init();
+                delegate = init();
             } catch (RuntimeException e) {
                 initException = e;
             }
-            return httpClient;
+            return delegate;
         }
     }
 
-    @Override
-    public void enhanceRequest(HttpPost request) {
+    public static void dispose(HttpResponse response) {
```

Comment:
why change this method to be static?

Reply:
to reflect better that it doesn't rely on any instance state